front/pages/api/v1/w/[wId]/spaces/[spaceId]/apps/[aId]/runs/index.ts

```diff
@@ -506,7 +506,7 @@ async function handler(
 }
 
 export default withPublicAPIAuthentication(
-  withResourceFetchingFromRoute(handler, "space"),
+  withResourceFetchingFromRoute(handler, { space: { requireCanRead: true } }),
```

Comment:
worth a comment here. canRead is for the apps which can be a public app. Runs can be created on public apps yet don't require canWrite on the apps'space, the run being created in the key's workspace.